### ISSUE TITLE
Unknown pattern `XR`

### ISSUE THREAD

**Issue by mimoo** [flynn/noise#21]
Hello,

What is this [XR pattern](https://github.com/flynn/noise/blob/master/patterns.go#L101:L109)?

```go
var HandshakeXR = HandshakePattern{
	Name: "XR",
	Messages: [][]MessagePattern{
		{MessagePatternE},
		{MessagePatternE, MessagePatternDHEE},
		{MessagePatternS, MessagePatternDHSE},
		{MessagePatternS, MessagePatternDHSE},
	},
}
```

I do not see any mention of it in the specification of Noise.

**Comment by titanous** [comment]
Looks like it was removed: https://github.com/noiseprotocol/noise_spec/commit/3da4183a05254a9f304749563971bf17e239fe81

**Comment by mimoo** [comment]
Alright, will push a change to include the initiator as well as removing the XR pattern if that's OK?

**Comment by titanous** [comment]
SGTM

**Comment by titanous** [comment]
Fixed by #22.
